typescript implementation/ascent-service/src/requestHelper.ts

```diff
@@ -0,0 +1,54 @@
+const request = require('request');
+import Request from "request";
+
+export interface responseFromDatabase
+{
+    code: number;
+    message: string;
+};
+
+export function sendPostRequest(endPoint: string, data: any, callback: Function) {
+    
+    let requestOptions = {
+        url: 'http://localhost:29000/' + endPoint,
+        body: JSON.stringify(data),
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }
+    new Promise<any>((resolve,reject) => {
```

Comment:
This is an odd way to use a promise. I'm not very familiar with promises myself, but for some reason this looks wrong. Maybe someone with more experience can also have a look at this and give their opinion.